File: src/Subscriber.cpp
#include "ros/ros.h"
#include "std_msgs/Float64.h"

void chatterCallback(const std_msgs::Float64::ConstPtr& msg)
{
  ROS_INFO("Count: [%f]", msg->data);
}

int main(int argc, char **argv)
{
  ros::init(argc, argv, "subscriber");
  ros::NodeHandle n;
  ros::Subscriber sub = n.subscribe("counter", 1, chatterCallback);
  ros::spin();

  return 0;
}

File: src/Subscriber.py
#!/usr/bin/env python3
import rospy
from std_msgs.msg import Float64

def funcion(msg): # Tipo callback
  resultado = msg.data  
  print(resultado)

if __name__ == "__main__":

  rospy.init_node('subscriber')
  rospy.Subscriber('counter', Float64, funcion)
  rospy.spin()
